src/js/models/v2/awardsV2/BaseFinancialAssistance.js

```diff
@@ -54,7 +54,8 @@ BaseFinancialAssistance.populate = function populate(data) {
     if (data.period_of_performance) {
         const periodOfPerformanceData = {
             startDate: data.period_of_performance.start_date,
-            endDate: data.period_of_performance.end_date
+            endDate: data.period_of_performance.end_date,
+            lastModifiedDate: data.period_of_performance.last_modified_date
         };
```

Comment:
I feel like this should be in the CoreAward Model? Would like to hear your justification for putting it here before approving.

Reply:
I put it here because that is where it is for contracts and that is where the other dates are housed.

Reply:
NVM looking at the code I see what is going on.